### DISCUSSION TITLE
Can we do batch read in Supabase?

### DISCUSSION THREAD

**Discussion by RatulSaha** [supabase/supabase#3646] [Questions] [answered]
Newbie here. I have multiple select/eq requests that I was hoping to batch to reduce round trips (and maybe database read time). Something like

```
await supabase
        .from("table")
        .select("*")
        .batcheq[("column", val1), ("column", val2), ("column", val3)]
```

Is it possible to achieve this with RPC or some other way?

**Comment by GaryAustin1** [comment, marked as answer]
`.in('column', [val1, val2, val3])`

**Reply by soedirgo** [reply]
It's not possible to batch multiple requests, but the above should do what you want @RatulSaha.

**Reply by RatulSaha** [reply]
Thank you, @GaryAustin1 @soedirgo!
